etc/supervisor/conf.d/vta.otp.conf

```diff
@@ -0,0 +1,15 @@
+[program:vta_otp]
+command=java -Xmx6G -Xverify:none -jar /srv/tripplanner/otp/otp-1.2.0-shaded.jar --basePath /srv/tripplanner/data/ --router default --server --port 8888 --cache /srv/tripplanner/otp/ned --autoReload --autoScan
```

Comment:
:cool: :+1: Nice that looks about right from what I recall when I built with Maven, so we got 6 GB reserved, assuming you've tested all this on our :green_heart: staging server, I'm good to approve and merge.